apps/finance/contracts/Finance.sol

```diff
@@ -94,7 +93,7 @@ contract Finance is AragonApp, EtherTokenConstant {
     // is changed before the operation if needed
     modifier transitionsPeriod {
         bool completeTransition = tryTransitionAccountingPeriod(getMaxPeriodTransitions());
-        require(completeTransition);
+        require(completeTransition, "Incomplete transition");
```

Comment:
I wouldn't add revert reasons for now, as hardcoding strings in the contract is going to increase the deployment gas costs enormously. I'd say let's not do it for now and think what the best way to do it would be. This is a good start: https://ethereum-magicians.org/t/building-a-taxonomy-of-smart-contract-errors/1170/7